spec/features/admin_add_food_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+RSpec.feature "Foods", type: :feature do
+  before(:each) do
+    @user = User.create(first_name: "Toyosi",
+                        last_name: "Famakinde",
+                        email: "toyosif@yahoo.com",
+                        password: "jeffrules",
+                        password_confirmation: "jeffrules",
+                        role: "admin")
+
+    Category.create(title: "Entree", description: "first meal")
+    @F1 = Food.create(name: "Spaghetti",
+                      description: "Best spaghetti ever",
+                      price: 1500.00 )
+    visit root_path
+    click_link('Log in')
+    expect(current_path).to eq(login_path)
+    expect(page).to have_selector("h3", text: "Log in")
+    fill_in "session_email", with: 'toyosif@yahoo.com'
+    fill_in "session_password", with: 'jeffrules'
+    click_button('Log in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.